footer.php

```diff
@@ -1,3 +1,6 @@
+		<?php if (wp_installing()) : ?>
```

Comment:
Consider adding a comment here (and in header.php) explaining why this container is added